scripts/makeLegacyPlots_postFit.py

```diff
@@ -493,21 +493,28 @@ def scaleStatSystUncertaintyBandForStack (grUncert,bkgSum):
     ######################### START DOING ACTUAL  PLOTTING STUFF ####################
     doOverflow = args.overflow
 
-    hEWK     = getHisto("EWK", hBkgs, doOverflow)
-    hggH     = getHisto("ggH_htt", hBkgs, doOverflow)
-    hTTH     = getHisto("ttH_hbb", hBkgs, doOverflow)
-    hTTX     = getHisto("TTX", hBkgs, doOverflow)
-    hVVV     = getHisto("VVV", hBkgs, doOverflow)
-    hTW      = getHisto("TW", hBkgs, doOverflow)
-    hTT      = getHisto("TT", hBkgs, doOverflow)
-    hQCD     = getHisto("QCD", hBkgs,doOverflow)
-    hWH      = getHisto("WH_htt", hBkgs, doOverflow)
-    hsingleT = getHisto("singleT", hBkgs, doOverflow)
-    hWJets   = getHisto("W", hBkgs, doOverflow)
-    hqqH     = getHisto("qqH_htt", hBkgs, doOverflow)
-    hVV      = getHisto("VV", hBkgs, doOverflow)
-    hDY      = getHisto("DY", hBkgs, doOverflow)
-    hZH      = getHisto("ZH_hbb", hBkgs, doOverflow)
+    gData = getHisto("data", hDatas , doOverflow) # NB - data is stored as a TGRaphAsymErrors inside a ROOFit object
+    gDataNonScaled = gData.Clone("gDataNonScaled")
+
+    # dummy empty TH1F to avoid problems when histos are not present in the fitdiagnostic.root file (we argued that if they are not there it is because they are empty)
```

Comment:
`we argued` --> `we agreed`

Reply:
In this context 'argued' is the correct verb, cause we are not sure if it is the truth but we are still supporting it as our best theory ;)

Reply:
ok I see!